### ISSUE TITLE
Once website is ready then use data highlighter google web master tools

### ISSUE THREAD

**Issue by mattweevu** [SolveSmartCities/SolveSmartCities.App#163]
This includes blog and such up and running on website. New language on website.

**Comment by reidcompton** [comment]
Which data set specifically do you want to use the highlighter for?


--
Reid Compton

727.244.7420
reid@reidcompton.com
reidcompton.com

**Comment by reidcompton** [comment]
duplicate of #123 (sitemap is in list of recommended actions)
